project.md

```diff
@@ -11,41 +11,96 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+People often brainstorm ideas by writing down whatever comes to mind and organizing them later. This can be students trying to find ideas and sources for an essay, travelers planning a trip to some destination, or an all-purpose note sheet for random thoughts and to-dos. However, the result of brainstorming is often hard to parse because of information overload. Nonetheless, there is generally some existing organization within the brainstormed document and a desired organization of these ideas post-brainstorming. An example would be turning potential places to visit into a formal itinerary for the traveler's case.
+
+At this moment, there are usually pre-existing file templates that people can copy and paste their ideas or apps that walk users through creating a document with some structure from scratch. However, neither of these solutions helps users sift through the information they already have - meaning they might potentially have forgotten or missed details that seem trivial but are actually important.
+If I could help them, I want to give users a tool to do a coarse re-ordering of their documents so that those hidden connections may become more visible. I believe this tool should not be a substitute for those templates or apps mentioned earlier. Instead, users should be able to use this tool quickly so that the later parts of the planning process (whether for an itinerary, essay, or some other format) can be smoother.
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL for this scenario is appropriate because it can be an interface to quickly sift through the content without the fatigue from manual editing. A general API can also do the same, but since the aim is to automate a task in the writing process, a DSL can be more easily designed so that users do not have to context-switch to a “programming” style of thinking and helping people stay “in the zone” of writing. Additionally, a DSL can be designed to be more declarative, which could be more intuitive for non-programmers to use.
+
+This need is domain-specific - it focuses on a subset of word documents that have an implicit or designed order and modules that can be identified by keywords. The limited expressiveness is also guaranteed because this task is focused on reordering existing data, rather than editing the data itself.
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+I interviewed my high school friend (who works in sales/marketing) about her experience with computers. She recently booked a flight for a trip and ran into a lot of issues with booking flights. Our conversation then verged into planning for travel, which is how this idea came about.
+
```

Comment:
When we did the activity in class that required us to go out on the HMC campus, I interviewed prof Bang who told me that he desired something similar. He told me that he had a lot of information scattered in various forms and places (in books, on his computer, in notebooks...) that he would love to collected in one place. He said that no current information organization system meet his needs because none of them allow him to keep track of different types of information. This may be something you keep in mind when you add features to your DSL and start to create your syntax